packages/react/package.json

```diff
@@ -8,7 +8,7 @@
   "license": "MIT",
   "devDependencies": {
     "@types/react-dom": "^16.9.5",
-    "@types/react-reconciler": "^0.18.0",
+    "@types/react-reconciler": "*",
```

Comment:
Why the loosening of this version constraint?

Reply:
this dependency is very far behind (should be `0.25.0`), also, the version for `react-recompiler` is `*`